frontend/src/components/Sidebar.vue

```diff
@@ -33,7 +48,7 @@
         </button>
       </div>
 
-      <div>
+      <div v-if="user.perm.admin">
```

Comment:
Why if that added?

Reply:
Because the `toSettings` function now navigates to `settings/global`, which is only accessible to an admin right?